config/locales/de.yml

```diff
@@ -215,9 +215,50 @@ de:
   activerecord:
     errors:
       messages:
-        <<: *errors_messages
+        inclusion: "ist kein gültiger Wert"
+        exclusion: "ist nicht verfügbar"
+        invalid: "ist nicht gültig"
+        confirmation: "stimmt nicht mit der Bestätigung überein"
+        accepted: "muss akzeptiert werden"
+        empty: "muss ausgefüllt werden"
+        blank: "muss ausgefüllt werden"
+        too_long: "ist zu lang (nicht mehr als {{count}} Zeichen)"
+        too_short: "ist zu kurz (nicht weniger als {{count}} Zeichen)"
+        wrong_length: "hat die falsche Länge (muss genau {{count}} Zeichen haben)"
+        taken: "ist bereits vergeben"
+        not_a_number: "ist keine Zahl"
+        greater_than: "muss größer als {{count}} sein"
+        greater_than_or_equal_to: "muss größer oder gleich {{count}} sein"
+        equal_to: "muss genau {{count}} sein"
+        less_than: "muss kleiner als {{count}} sein"
+        less_than_or_equal_to: "muss kleiner oder gleich {{count}} sein"
+        odd: "muss ungerade sein"
+        even: "muss gerade sein"
+        record_invalid: "Gültigkeitsprüfung ist fehlgeschlagen: {{errors}}"
       template:
         <<: *errors_template
-
       full_messages:
         format: "%{attribute} %{message}"
+
+  mailer:
+    general:
+      footer:
+        general:
+          html: |-
+              <p>
+               Wenn diese E-Mail irrtümlich an dich geschickt wurde, melde das bitte dem <a href="%{support_url}" style="color:#ffffff; text-decoration:underline">Support</a>.
+              </p>
+
+              <p>
+              Das Archive of Our Own - AO3 (Ein Eigenes Archiv) wird von Fans unterhalten und ist deshalb auf <a href="%{donate_url}" style="color:#ffffff; text-decoration:underline">deine Spenden</a> angewiesen.
+              </p>
```

Comment:
The German, Spanish and French yml files don't seem to have as many strings as the English one - `sentat` and `sendby` are missing. I don't know if this is deliberate though.